extensions/amp-analytics/0.1/events.js

```diff
@@ -507,6 +507,34 @@ export class VisibilityTracker extends EventTracker {
   }
 }
 
+export class ErrorTracker extends EventTracker {
```

Comment:
i thought you're thinking of recusing `CustomEventTracker` ? What's the final decision?